worlds/rogue_legacy/__init__.py

```diff
@@ -42,144 +42,142 @@ class RLWorld(World):
 
     item_pool: List[RLItem] = []
```

Comment:
This is potentially bad. Shouldn't be initialized here.

Reply:
No longer needed anyway. Moved out.